ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -123,6 +123,7 @@ Item {
                     id: filterButton
                     icon.name: "filter"
                     checkable: true
+                    visible: walletTabBar.currentIndex !== 2 // hide filter button on activity tab
```

Comment:
This is not part of the fix and not even desired:

![image](https://github.com/status-im/status-desktop/assets/5377645/cecf6694-62a5-491a-b5b3-35678cee96c8)